charts/private-action-runner/README.md

```diff
@@ -0,0 +1,47 @@
+# Private action runner Helm chart
+
+This Helm Chart will deploy the Datadog Private Action runner inside a Kubernetes cluster.
+You will be able to use private actions from the Datadog Workflow and Datadog App Builder products.
+When deploying this chart, you will be able to give permissions to the runner in order to be able to run Kubernetes actions.
+
+## Requirements
+* A Datadog account with private actions enabled
+* The kubectl cli
+* Helm
+* Sufficient permissions to the kubernetes cluster
+
+## In order to use this chart
+
+* Go to the private action runner tab https://app.datadoghq.com/workflow/private-action-runners
+* Create a new private action runner
+* Follow the instructions and you should have a running docker container and `config/config.yaml` file
+* Stop the docker container (`docker stop <name-of-the-container>` or `docker compose stop`)
+* Replace the `URN_FROM_CONFIG` and the `PRIVATE_KEY_FROM_CONFIG` from the chart's `values.yaml` with the `urn` and the `privateKey` from the `config/config.yaml` of the docker container.
+* Create the kubernetes namespace for your runner `kubectl create namespace private-action-runner`
+* Replace the `namespace` in the `values.yaml` with the namespace you just created
+* Install the Helm chart `helm install <name> .`
```

Comment:
I think we can reference the chart public URL ? But customers should have other instructions on how to pass the values

Reply:
I saw synthetics has https://github.com/DataDog/helm-charts/blob/bc1b5bb922e68106f1b6e222bd2b7e90a90d351b/charts/synthetics-private-location/README.md.gotmpl#L11C1-L23C1
so I reworked our instructions to be more similar to theirs wdyt?

Reply:
agreed, I renamed it to `config.yaml`